pkg/apb/client.go

```diff
@@ -200,8 +194,5 @@ func (c *Client) refreshLoginToken(clusterConfig ClusterConfig) error {
 	if err != nil {
 		return err
 	}
-
-	c.log.Debug("No error reported after running oc login. Cmd output:")
-	c.log.Debug(string(output))
```

Comment:
If the oc login errors it will return the error.  Otherwise the output is the just a block of text from an oc login.  Either way we end up with the error message or we print success on line 158.